pom.xml

```diff
@@ -20,10 +20,10 @@
     <description>Mule connector lets you easily send and receive messages to queues and topics for any message service which implements the JMS specification</description>
 
     <properties>
-        <activemq.version>5.15.16</activemq.version>
+        <activemq.version>5.16.7</activemq.version>
         <artemisVersion>2.25.0</artemisVersion>
         <javaxJmsApiVersion>2.0.1</javaxJmsApiVersion>
-        <commonsIoVersion>2.11.0</commonsIoVersion>
+        <commonsIoVersion>2.14.0</commonsIoVersion>
```

Comment:
Can we update to 2.15? because NexusIQ shows 2.14.0 with vulnerabilities:
<img width="1406" alt="image" src="https://github.com/user-attachments/assets/45d0811a-27d8-4282-8e40-b19306e56d05" />

<img width="903" alt="image" src="https://github.com/user-attachments/assets/ceff4712-c36f-4d51-a8bb-f60d941fea8a" />